Extensions/tweaks.js

```diff
@@ -718,8 +718,15 @@ XKit.extensions.tweaks = new Object({
 			}
 		}
 
-		if (XKit.extensions.tweaks.preferences.hide_follows.value) {
-			XKit.extensions.tweaks.add_css(".note a.follow {display: none !important; } .notification_follow, .reblog_follow_button { display: none !important; } #posts .notes_outer_container.popover .note a.follow { display: none !important; } #posts .notes_outer_container.popover .note.like a.block { right: 16px; }", "xkit_tweaks_hide_follows");
+		if (XKit.extensions.tweaks.preferences.hide_follows.value && XKit.page.react) {
+			let followButtonSelectors = XKit.css_map.keyToClasses('followButton');
+			let postSelectors = XKit.css_map.keyToClasses('post');
+			let postFollowButtonSelectors = postSelectors.map(postCssClass => {
```

Comment:
flatMap might make this a little clearer (and I think the browser support is there? I haven't checked recently though)

Reply:
Actually, I think there's a helper for this now? check xkit_patches, right under where keyToClasses is defined